modular_skyrat/modules/indicators/code/combat_indicator.dm

```diff
@@ -71,6 +98,31 @@ GLOBAL_VAR_INIT(combat_indicator_overlay, GenerateCombatOverlay())
 		return
 	set_combat_indicator(!combat_indicator)
 
+/obj/vehicle/sealed/proc/handle_ci_migration(mob/living/user)
+	if(!typesof(user.loc, /obj/vehicle/sealed))
+		return
+	if ((src.max_occupants > src.max_drivers) && ((!(user in return_drivers())) && (src.driver_amount() > 0)))
+		return
+	if (user.combat_indicator && !combat_indicator_vehicle)
+		combat_indicator_vehicle = TRUE
+		update_appearance(UPDATE_ICON|UPDATE_OVERLAYS)
+
+/obj/vehicle/sealed/proc/disable_ci(mob/living/user) //as of writing this, 12/16/2021: something is calling procs that use this twice. dont know why or where
```

Comment:
If it's being called twice, use a break point and use the stack trace in the debug tab of VSC to figure out what's calling it both times. If you need help figuring that out, I can help you.

Also documentation :)